src/renderer/containers/Table/CustomCells/StatusCell/index.tsx

```diff
@@ -60,12 +60,13 @@ export default function StatusCell(props: CellProps<UploadSummaryTableRow>) {
         <CheckCircleFilled className={styles.success} />
       );
     } else {
-      // Switch the incomplete status tooltip depending on if the post upload
-      // has yet to run vs if it has failed
-      if (etlProcess?.status === JSSJobStatus.FAILED) {
-        tooltip = `${tooltip} - File has been successfully uploaded to FMS, but may not be viewable in the FMS File Explorer. Attempt to make it visible in the FMS Explorer resulted in the following error: ${etlProcess.status_detail}`;
+      if (
+        props.row.original.serviceFields?.postUploadProcessing?.etl?.status ===
+        JSSJobStatus.FAILED
+      ) {
+        tooltip = `${tooltip} - File has been successfully uploaded to FMS, but may not be viewable in the File Upload App. Attempt to make it visible in the FMS Explorer resulted in the following error: ${props.row.original.serviceFields?.postUploadProcessing?.etl?.status_detail}`;
       } else {
-        tooltip = `${tooltip} - File has been successfully uploaded; working on making it visible in the FMS File Explorer if it isn't already`;
+        tooltip = `${tooltip} - File has been successfully uploaded; working on making it visible in the File Upload App if it isn't already`;
```

Comment:
The File Upload App is making an uploaded file visible in the File Upload App?

Reply:
Well it _does_ do that, it does make uploaded files visible in the FUA itself after upload, but looking at this again I'm honestly not sure which is more correct for this message...